pkgs/tools/package-management/nixpkgs-review/default.nix

```diff
@@ -23,6 +24,14 @@ python3.pkgs.buildPythonApplication rec {
     sha256 = "sha256-9fdoTKaYfqsAXysRwgLq44UrmOGlr5rjF5Ge93PcHDk=";
   };
 
+  patches = [
+    (fetchpatch {
+      name = "fix-detection-of-broken-packages.patch";
```

Comment:
Release done in https://github.com/Mic92/nixpkgs-review/releases/tag/2.9.2

Reply:
thanks

opened #233728